artman/cli/support.py

```diff
@@ -116,3 +116,33 @@ def resolve(name, user_config, flags, default=None):
     if getattr(flags, name, None):
         answer = getattr(flags, name)
     return answer
+
+
+def select_git_repo(git_repos, target_repo):
```

Comment:
I like this move, but make sure the unit tests move also. (The tests directory structure should mirror the code structure. That is not done perfectly right now, but it is done correctly in that portion, and I would like to maintain that.)